### DISCUSSION TITLE
[Bug] No option to rename a stack

### DISCUSSION THREAD

**Discussion by anthonysmillie** [louislam/dockge#22] [Feature Request]
When editing a stack there's no option to rename the stack

**Comment by anthonysmillie** [comment]
Temporary workaround for this: Rename the stack's folder to your desired name, IE:

```bash
mv /opt/stacks/old_name /opt/stacks/new_name
```

And then re-scan your stacks directory.

**Reply by yewkay** [reply]
Can we put this in as a feature request? I pasted a docker compose file and started a container.  Unknowingly, the service became known as "db" now because the first service was db. Now it's called db permanently.
